### DISCUSSION TITLE
you should not provide watch options in `webpack.config` then, use them in `compiler.watch` as an argument

### DISCUSSION THREAD

**Discussion by ygs-code** [webpack/webpack#15916] [General]
you should not provide watch options in `webpack.config` then, use them in `compiler.watch` as an argument

_Originally posted by @vankop in https://github.com/webpack/webpack/issues/15890#issuecomment-1144726957_

What if I want to get the compilation logs in my plug-in?

**Comment by alexander-akait** [comment]
Please ask this in questions, no need to create issue for this, thank you

**Reply by alexander-akait** [reply]
You should use `watchOptions` for `webpack.watch(options)`, you can use `webpack-cli`, so we will do it for you
